models/staging/stg_amazon_selling_partner__item_image.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(enabled=var('amazon_selling_partner__using_catalog_module', true)) }}
+
+{% set base_table = ref('stg_amazon_selling_partner__item_image_base') if var('amazon_selling_partner_sources',[]) != [] else source('amazon_selling_partner', 'item_image') %}
+
+with base as (
+
+    select * 
+    from {{ base_table }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(base_table),
+                staging_columns=get_item_image_columns()
+            )
+        }}
+        
+        {{ amazon_selling_partner_apply_source_relation() }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        source_relation, 
+        _fivetran_synced,
+        asin,
+        height,
+        link,
+        marketplace_id,
+        upper(variant) as variant,
```

Comment:
Is there a reason we went with upper when we've taken a lower approach before? We should standardize this across this package as well as our approach across the board.

Reply:
I went with upper to match the values in the [source](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#variant) but we do typically lean toward `lower`. Any thoughts?